packages/office-ui-fabric-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "office-ui-fabric-react",
-  "version": "7.117.1",
+  "version": "7.117.2",
```

Comment:
did you get this updated by running `beachball sync`? i forgot if we also need to sync the places where have oufr as dependency

Reply:
Yes that was `beachball sync`. It didn't update anything else, but I think that should be okay in this case.